aws-datastore/src/main/java/com/amplifyframework/datastore/AWSDataStorePlugin.java

```diff
@@ -190,12 +190,14 @@ public AWSDataStorePlugin(@NonNull ModelProvider modelProvider) {
      */
     @Deprecated
     @VisibleForTesting
-    AWSDataStorePlugin(@NonNull ModelProvider modelProvider, @NonNull ApiCategory api) {
+    AWSDataStorePlugin(@NonNull ModelProvider modelProvider,
+                       @NonNull ApiCategory api,
+                       @Nullable DataStoreConfiguration dataStoreConfiguration) {
         this(
             Objects.requireNonNull(modelProvider),
             ModelSchemaRegistry.instance(),
             Objects.requireNonNull(api),
-            null
+            dataStoreConfiguration
```

Comment:
nit: maybe use the builder here instead.

Reply:
It turns out when I rebased with main, I no longer have changes in this file.